### ISSUE TITLE
region_settings overwrites all previous region_overlays

### ISSUE THREAD

**Issue by Light-Wave** [CleverRaven/Cataclysm-DDA#60394]
### Describe the bug

If you load a mod with a region_overlay, and then load a mod with a region_settings, the region_overlay gets completely discarded.

### Steps to reproduce

Edit the "field" field in mods/DinoMod/regional_overlay.json to read `"field": { "chance": 1, "extras": { "mx_mass_grave_dino": 300000 } }`
Create a world where you add DinoMod first, and TropiCataclysm second
Boot up the world and notice there are no dino mass graves.
Create a world with TropiCataclysm first and DimoMod second
Boot up the world and notice there are dino mass graves on most every field tile.

### Expected behavior

The player should not have to manually manage load order in order to make sure the mods they're playing with works. Or if they do need to do so, they should be given some clues as to what order the mods have to be in.

region_overlays should probably be applied after all region_settings have finished being applied. (IMHO the preferred solution)
Alternatively, mods with a region_settings could be forced to load before mods with a region_overlay

### Screenshots

![mass_grave](https://user-images.githubusercontent.com/66904273/186133770-7d6502c0-76ec-40dc-bd70-965594faa6ed.PNG)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19042.1110 (20H2)
- Game Version: 630e438 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    DinoMod [DinoMod],
    TropiCataclysm [tropicata]
]

**Comment by ZhilkinSerg** [comment]
Indeed, region overlays are loaded and applied before region settings from another mod are loaded:

![image](https://user-images.githubusercontent.com/16213433/186256063-8a911bdc-c729-4037-a331-517dbef9ced8.png)
